nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestListenUDP.java

```diff
@@ -47,7 +44,9 @@
 import java.util.List;
 import java.util.concurrent.BlockingQueue;
 
-@RunWith(MockitoJUnitRunner.class)
```

Comment:
It looks like this class needs to be annotated with `@ExtendWith(MockitoExtension.class)` in order to initialized member variables annotated with `@Mock`.

Reply:
Thanks for the tip! Making the changes